Microsoft.Research/ManagedContract.Setup/DotNet.Contracts.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <CodeContractsInstallDir>$(MSBuildThisFileDirectory)</CodeContractsInstallDir>
+  </PropertyGroup>
+
+  <!-- Use MsBuild specific targets file if availible -->
+  <Import Project="$(CodeContractsInstallDir)\MsBuild\v$(MSBuildToolsVersion)\Microsoft.CodeContracts.targets"
```

Comment:
Seems like I missed to change it. It was originally VisualStudioVersion from another .targets file but MSBuildTools seemed correct. Will fix